### PR TITLE
Massive overhaul.

### PR DESCRIPTION
**Pull request by Kazkin**
Redoes the economy, much of the map, and fixes various bugs and introduces new content. A combined total of 26 hours of work.

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/secure/militia.dm [hunk 1]
```diff
@@ -220,6 +197,7 @@
 	new /obj/item/weapon/storage/firstaid/adv(src)
 	new /obj/item/weapon/storage/firstaid/ifak(src)
 	new /obj/item/weapon/storage/pill_bottle/tramadol(src)
+	/obj/item/weapon/gun/projectile/boltgun
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/crates_lockers/closets/secure/militia.dm:200]:
```suggestion
	new /obj/item/weapon/gun/projectile/boltgun(src)
```

File: code/modules/projectiles/guns/projectile/boltgun.dm [hunk 2]
```diff
@@ -234,7 +234,7 @@
 	icon_state = "starstriker"
 	item_state = "starstriker"
 	max_shells = 10
-	price_tag = 500 //no...
+	price_tag = 250 //no...
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/boltgun.dm:237]: This is already heavily lowered from a normal guns price do to factorial memes

**Review by Trilbyspaceclone — CHANGES_REQUESTED** [review]
Just one code based issue

File: code/modules/projectiles/guns/projectile/boltgun.dm (hunk 2 above)
Comment (Kazkin) [reply code/modules/projectiles/guns/projectile/boltgun.dm:237]: Every gun got reduced by 50% so this brings it in line with all other firearms.

**Pull request merged**